angularjs-1.5-basics/gulp-tasks/build.js

```diff
@@ -0,0 +1,247 @@
+'use strict';
+
+var gulp = require('gulp');
```

Comment:
__[jshint]__
* 'require' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->